README.md

```diff
@@ -80,6 +80,20 @@ cargo update -p indexmap --precise "2.5.0"
 cargo update -p security-framework-sys --precise "2.11.1"
 ```
 
+## Troubleshooting Tests
+
+If tests are failing with `cargo test` it could be an issue related to tests that use bitcoind or electrsd running in parallel. To force tests to run in series you can try:
+
```

Comment:
Do you know if anyone else faced this on Linux? I guess it's a macOS-related issue, maybe we should point that out here?

Reply:
I am having this issue on mac, however I have no idea if it's exclusive to mac so it's probably worth being aware of for anyone who's having issues running the tests

Reply:
went ahead and added a note that it's known to affect mac users